### PR TITLE
Fix errors

### PR DESCRIPTION
**Pull request by saood06**
Added some strings, and tests, and some other minor fixes

### CODE REVIEW

**Comment by saood06** [comment]
I just found potentially another issue (can't reproduce it consistently), but sometimes a  Youtube(NotFound) will be thrown for videos that are definitely available (a second pass at them should find them (might require a bit of a cooldown not sure).

File: src/youtube/next.rs [hunk 1]
```diff
@@ -37,26 +37,30 @@ pub struct Results2 {
...
     }
 
     pub fn secondary(&self) -> &VideoSecondaryInfoRenderer {
-        if let Content::VideoSecondaryInfoRenderer(ret) = &self.contents[1] {
+        let index = self.contents.len() - 2;
+        if let Content::VideoSecondaryInfoRenderer(ret) = &self.contents[index] {
             ret
         } else {
-            unreachable!("VideoSecondaryInfoRenderer was not at index 1")
+            dbg!(self.contents.len());
```
Comment (ATiltedTree) [inline src/youtube/next.rs:53]: This probably shouldn't be here, right?

**Review by ATiltedTree — CHANGES_REQUESTED** [review]
One minor thing, otherwise it looks good to me.

File: src/youtube/next.rs (hunk 1 above)
Comment (saood06) [reply src/youtube/next.rs:53]: Sorry, your right.

Comment (saood06) [reply src/youtube/next.rs]: Pushed a commit to remove it

**Comment by ATiltedTree** [comment]
Thanks!

**Comment by saood06** [comment]
> Thanks!

Nah thank you for the library, I felt bad about the 2 issues I opened so I fixed whatever I found myself

**Comment by ATiltedTree** [comment]
Well, I am happy that my library is useful to someone!

**Pull request merged**